### DISCUSSION TITLE
Maplibre or Mapbox?

### DISCUSSION THREAD

**Discussion by alcy** [keplergl/kepler.gl#2622] [Q&A] [answered]
Hello! Thanks for developing kepler.gl. I would like to know if kepler.gl uses a mix of Maplibre and Mapbox, or just Maplibre? The introduction section in the documentation talks about kepler.gl using Maplibre, but the getting started sections talk about Mapbox. Can you please clarify which one is currently recommended for development? Thanks!

**Comment by ibgreen** [comment, marked as answer]
@alcy kepler.gl 3.0 transitioned from Mapbox to Maplibre, see e.g this [blog post](https://www.linkedin.com/pulse/introducing-keplergl-30-location-foursquare-wphac/).

Unfortunately, the kepler.gl documentation is out of date in a number of places. We appreciate any PRs to help us improve :)

**Reply by alcy** [reply]
Thanks a lot @ibgreen, I would certainly try to get myself up to speed with the current way of doing things and then sending PRs wherever applicable! :) :)
